### ISSUE TITLE
Does this still work with Angular 4?

### ISSUE THREAD

**Issue by austinthedeveloper** [davguij/angular-npm-module-seed#10]
I've been messing around with this project and no matter what I do, I'm getting this error when importing to my Angular 4+ project:

`Unexpected value 'CountToModule' imported by the module 'AppModule'. Please add a @NgModule annotation.`

I've tried getting this to work 2 different ways:

- Build the original project with no modifications, link, and import the module into the project 
- Update the Angular packages to `>=4.0.0`, typescript to `~2.3.3`, and reflect-metadata to `^0.1.10`. The build succeeds. Run `npm link` on the `dist` folder.

Both are imported in the project the same way:
`import { CountToModule } from 'angular-count-to';`
```
@NgModule({
  ...
  imports [CountToModule]
  ...
});
```

Tslint-wise, there's no error because it's imported correctly. But as soon as I run `ng-serve`, I get the error above.

Are there additional instructions on how to import a module?

**Comment by alubeck** [comment]
Yeah, same issue here! I watched this issue while using node > version 8. WIth 7.9 all seems fine. What was your node setup @austinthedeveloper ?

**Comment by austinthedeveloper** [comment]
I'm pretty sure it has to do with differences between Angular 2 and 4. I ended up going with another starter that does basically the same thing as this repo but works with 4. When I copy/pasted my exact code over to the new repo, it all worked.

**Comment by davguij** [comment]
Thanks for reporting, @austinthedeveloper and @alubeck. This seed project is outdated indeed and probably some maintenance work needs to be done on my side for the new Angular / Node versions

**Comment by zt1983811** [comment]
@austinthedeveloper Could you send me the link of the other project starter repo please? Thanks

**Comment by austinthedeveloper** [comment]
https://github.com/robisim74/angular-library-starter

I've built 2 modules with this one.
